src/test/java/hw5/steps/AssertionSteps.java

```diff
@@ -0,0 +1,92 @@
+package hw5.steps;
+
+import hw5.context.TestContext;
+import hw5.pages.MainPage;
+import hw5.pages.ServicePage;
+import hw5.pages.UserTablePage;
+import io.cucumber.java.en.Then;
+
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class AssertionSteps {
+
+    MainPage mainPage = new MainPage(TestContext.getInstance().getDriver());
+    ServicePage servicePage = new ServicePage(TestContext.getInstance().getDriver());
+    UserTablePage userTablePage = new UserTablePage(TestContext.getInstance().getDriver());
+
+    @Then("Browser title is equal to {string}")
+    public void browser_title_is_equal_to(String string) {
+        assertEquals(mainPage.getPageTitle(), string);
+    }
+
+    @Then("Correct username {string} is shown on the Main Page")
+    public void correct_username_is_shown_on_the_Main_Page(String string) {
+        assertEquals(mainPage.getPageUserName(), string);
+    }
+
```

Comment:
Please provide more clear names for the method input parameters

Reply:
Done.